src/language/collections.md

```diff
@@ -3,6 +3,9 @@ title: Collections
 description: Summary of the different types of collections in Dart.
 ---
 
+Lists, sets, and maps are all collections in Dart. You can learn more about each
+in the [Collections][] section of the Library tour, and on the [Generics][] page.
```

Comment:
This text seems to direct users away from this page to learn about collections. Could we provide some more specific context on why/when they would visit those pages? Or move the links to some sort of "Other resources" header at the end of the page.

For example, could state something like "To learn more about the underlying collection types and interacting with them, check out the [Collections][] section of the Library tour.".

Reply:
Oh I see you moved this from each section. I think I might prefer at least the specific library tour sections be linked to in the appropriate section. Since users may be reading the "map" section specifically and want to learn about the library usages. They may not think to find this at the top of page, and this also loses the specific section links.

At least as of now, each collection type isn't its own entry in the library tour TOC either, so they may have trouble finding it:

<img width="140" alt="Collections in library tour tOC" src="https://user-images.githubusercontent.com/18372958/231608358-69e2ec3f-0c79-4213-a9a3-f4aac3f20244.png">

Reply:
Ok great, I was thinking the same thing. I'll move a link to each types library section back into their language sections.